go/cmd/dolt/commands/diff.go

```diff
@@ -178,7 +164,7 @@ func (cmd DiffCmd) validateArgs(apr *argparser.ArgParseResults) errhand.VerboseE
 
 	f, _ := apr.GetValue(FormatFlag)
 	switch strings.ToLower(f) {
-	case "tabular", "sql", "":
+	case "tabular", "sql", "json", "":
```

Comment:
line 128 for the `-r` arg documentation could use an update to mention the new `json` param it accepts.

Reply:
Good catch!